canonical-web-and-design/ubuntu.com
# K8s install page refresh

## Done

- K8s install page refresh

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes/install
- Check the copy matches what is on the copy doc

## Issue / Card

Fixes #https://github.com/canonical-web-and-design/web-squad/issues/3984
